src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,68 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int MAXIMUM_SIZE = 10;
+    private Pair<K, V>[] storage;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
0 - default  value for int